web/src/lib/components/admin-page/jobs/jobs-panel.svelte

```diff
@@ -110,6 +110,12 @@
       allowForceCommand: false,
       component: StorageMigrationDescription,
     },
+    [JobName.Migration]: {
```

Comment:
I assume this job will be a temporary job, correct? We might be able to remove it after a few releases?

Reply:
The idea was that the Migration Job (and Queue respectively) will replace the storage template migration job at some point. So that Migration will be the more general form for everything file-related.

Reply:
Jason said that storage template migration could need a refactoring anyway, so they would be merged then